docs/tutorials/dataflows/nlp.rst

```diff
@@ -0,0 +1,89 @@
+Using NLP Operations
+==============================
+
+This example will show you how to use DFFML operations to clean text data and train a model using DFFML cli.
+
+DFFML offers several :ref:`plugin_models`. For this example
+we will be using the tensorflow DNNClassifier model
+(:ref:`plugin_model_dffml_model_tensorflow_tfdnnc`) which is in the ``dffml`` package.
+
+We will use two operations :ref:`plugin_operation_dffml_operations_nlp_remove_stopwords` and :ref:`plugin_operation_dffml_operations_nlp_get_embedding`.
+Internally, both of these operations use `spacy <https://spacy.io/usage/spacy-101>`_ functions.
+
+Operation `remove_stopwords` cleans the text by removing most commanly used words which give the text little or no information eg. but, or, yet, it, is, am, etc.
```

Comment:
Remember to add the description of each operation to the docstring for that operation as well